src/events/message/messageCreate.ts

```diff
@@ -26,8 +25,6 @@ export default class extends Event {
 		const context = await getCommandContext(message);
 		// checks if the user is in the blacklist
 		if (this.client.blacklist.get(context.author.id) !== undefined) return;
-
-		if (context.channel.type === "DM") return DMUtils(this.client, context);
```

Comment:
IMO we should still have a guard here. Without the intent, the API isn't _supposed_ to send us any DM events, but we should still have something to verify that.

Reply:
the next line takes care of that
`if (!context.guild || context.author.bot) return;`

Reply:
Got it